src/main/java/org/primeframework/mvc/cors/CORSConfiguration.java

```diff
@@ -38,10 +39,16 @@ public class CORSConfiguration implements Buildable<CORSConfiguration> {
 
   public boolean debug;
 
+  public Predicate<String> excludeUriPredicate;
```

Comment:
This should be named `excludeURIPredicate`, same for other names, `Uri` -> `URI`